tests/test_utils/test_search_utils.py

```diff
@@ -3,3 +3,56 @@
 Distributed under the MIT License
 (See an accompanying file LICENSE or a copy at https://opensource.org/licenses/MIT)
 """
+from typing import List
+
+from sc_client.constants import sc_types
+from sc_client.models import ScAddr
+
+from sc_kpm.utils.common_utils import create_node
+from sc_kpm.utils.generation_utils import wrap_in_oriented_set, wrap_in_set
+from sc_kpm.utils.search_utils import get_oriented_set_elements, get_set_elements, get_set_power
+from tests.common_tests import BaseTestCase
+
+
+class TestSearchUtils(BaseTestCase):
+    def test_get_set_elements(self):
+        set_node = create_node(sc_types.NODE_CONST)
+        elements = [
+            create_node(sc_types.NODE_CONST),
+            create_node(sc_types.NODE_CONST),
+            create_node(sc_types.NODE_CONST),
+        ]
+        wrap_in_set(set_node, *elements)
+        search_results = get_set_elements(set_node)
+        elements.reverse()
+        assert are_elements_equal(search_results, elements)
```

Comment:
Specify test, I want to be sure that construction forms correctly with concrete sequence